gui/wxpython/mapdisp/frame.py

```diff
@@ -1005,7 +1005,8 @@ def OnCloseWindow(self, event, askIfSaveWorkspace=True):
                             pgnum_dict=pgnum_dict, is_docked=self.IsDocked()
                         )
                         if not self.IsDocked():
-                            self.GetParent().Destroy()
+                            frame = self.GetParent()
+                            frame.closeFrameNoEvent()
```

Comment:
This is strange, why is this needed? Based on how closing windows work, the method closeFrameNoEvent should be equivalent to calling Destroy().

Reply:
right, it is not needed at all, I have replaced it by Destroy